src/components/CommentReply/index.tsx

```diff
@@ -78,6 +118,12 @@ export default class CommentReplyComponent extends React.Component<CommentReplyP
     );
   }
 
+  renderMessage(): React.ReactFragment {
+    const { reply } = this.props;
+
+    return <CommentMessage message={reply.message} />;
+  }
```

Comment:
Does this really need to be a function? Feels like you could just as well render the component directly. There are a lot of renderThing functions in this very long file that feel like they could just be separate components?